clang/test/CodeGenHLSL/builtins/ByteAddressBuffer-contructor.hlsl

```diff
@@ -0,0 +1,19 @@
+// RUN: %clang_cc1 -triple dxil-pc-shadermodel6.3-library -x hlsl -emit-llvm -disable-llvm-passes -o - %s | FileCheck %s --check-prefixes=CHECK,CHECK-DXIL
```

Comment:
Rename to ByteAddressBuffer**s**-constructor**s**.hlsl - test cases for other byte address buffers will be added here later on.